src/pairs/GameState.py

```diff
@@ -14,14 +15,37 @@ def __init__(self, deck, target_score, players):
             self.num_players = len(self.players)
 
     def deepcopy(self):
-        c = GameState(self.deck.deepcopy(), self.target_score, self.players.deepcopy())
+        c = GameState(self.deck.deepcopy(), self.target_score,
```

Comment:
nit: You can just return the GameState without needing another variable `c` in between.

Reply:
I like the c, so I'll keep it in, but thank you.